core_api/tests/conftest.py

```diff
@@ -28,38 +29,38 @@ def s3_client():
         )
     except ClientError as e:
         if e.response["Error"]["Code"] != "BucketAlreadyOwnedByYou":
-            raise e
+            raise
 
-    yield _client
+    return _client
```

Comment:
The idea here is that if you need to perform any teardown, you can yield the result and put the teardown code after the yield. If, on the other hand, you don't need any teardown of your own, you should just return object rather than yielding it.

Reply:
See [PT022](https://docs.astral.sh/ruff/rules/pytest-useless-yield-fixture/) for the Ruff rule which did this.